src/Modules/Build.lua

```diff
@@ -318,10 +318,10 @@ function buildMode:Init(dbFileName, buildName, buildXML, convertBuild)
 		{ stat = "AreaOfEffectRadius", label = "AoE Radius", fmt = "d" },
 		{ stat = "BrandAttachmentRange", label = "Attachment Range", fmt = "d", flag = "brand" },
 		{ stat = "BrandTicks", label = "Activations per Brand", fmt = "d", flag = "brand" },
-		{ stat = "ManaCost", label = "Mana Cost", fmt = "d", color = colorCodes.MANA, compPercent = true, lowerIsBetter = true, condFunc = function(v,o) return o.ManaHasCost end },
-		{ stat = "LifeCost", label = "Life Cost", fmt = "d", color = colorCodes.LIFE, compPercent = true, lowerIsBetter = true, condFunc = function(v,o) return o.LifeHasCost end },
-		{ stat = "ESCost", label = "Energy Shield Cost", fmt = "d", color = colorCodes.ES, compPercent = true, lowerIsBetter = true, condFunc = function(v,o) return o.ESHasCost end },
-		{ stat = "RageCost", label = "Rage Cost", fmt = "d", color = colorCodes.RAGE, compPercent = true, lowerIsBetter = true, condFunc = function(v,o) return o.RageHasCost end },
+		{ stat = "ManaCost", label = "Mana Cost", fmt = "d", color = colorCodes.MANA, pool = "Mana", compPercent = true, lowerIsBetter = true, condFunc = function(v,o) return o.ManaHasCost end, warnFunc = function(v,o) return (o.EnergyShieldProtectsMana and (v > (o.ManaUnreserved + o.EnergyShield)) or (v > o.ManaUnreserved)) and "You do not have enough Mana to use a Selected Skill" end },
```

Comment:
```suggestion
		{ stat = "ManaCost", label = "Mana Cost", fmt = "d", color = colorCodes.MANA, pool = "Mana", compPercent = true, lowerIsBetter = true, condFunc = function(v,o) return o.ManaHasCost end, warnFunc = function(v,o) return (not o.EnergyShieldProtectsMana and (v > o.ManaUnreserved) or (v > o.ManaUnreserved + o.EnergyShield)) and "You do not have enough "..(o.EnergyShieldProtectsMana and "Energy Shield and Mana" or "Mana").." to use a Selected Skill" end },
```
The issue here is that the following condition could be true if the left wasn't this switches the order to ensure we don't try the regular check on EB. Also improve tooltip.